src/App.jsx

```diff
@@ -1,3 +1,17 @@
+import { Route, Switch } from 'react-router-dom/';
+import LandingPage from './views/LandingPage';
+import Main from './views/Main';
+
 export default function App() {
-  return <h1>Hello World</h1>;
+  return (
+    <Switch>
+      <Route path="/char">
+        <LandingPage />
+      </Route>
+
+      <Route path="/">
+        <Main />
+      </Route>
```

Comment:
To place the home page as the list view, we can render the landing page component here and then use a nested route of /:id within the landing page.